project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ object Dependencies {
 
   val workbenchUtilV   = "0.2-72adc94"
   val workbenchModelV  = "0.4-72adc94"
-  val workbenchGoogleV = "0.5-72adc94"
+  val workbenchGoogleV = "0.5-58c3e7b-SNAP"
```

Comment:
Just noting when https://github.com/broadinstitute/workbench-libs/pull/36 is picked up the `workbenchGoogle` dependency will need to change to something like `... % "test->test;compile->compile"`.

Reply:
I tinkered with Leo until it worked and found that `"org.broadinstitute.dsde.workbench" %% "workbench-google"  % workbenchGoogleV % "test" classifier "tests"` worked. `test->test;compile->compile` didn't, I don't think